src/level_gen.cpp

```diff
@@ -124,16 +124,20 @@ static inline Entity createAgent(Engine &ctx, const MapObject &agentInit) {
 
 static Entity makeRoadEdge(Engine &ctx, const MapVector2 &p1,
                            const MapVector2 &p2, const EntityType &type) {
+
+                    
     float x1 = p1.x;
     float y1 = p1.y;
     float x2 = p2.x;
     float y2 = p2.y;
 
-    Vector3 start{.x = x1 - ctx.data().mean.x, .y = y1 - ctx.data().mean.y, .z = 1};
-    Vector3 end{.x = x2 - ctx.data().mean.x, .y = y2 - ctx.data().mean.y, .z = 1};
+    float z = 1 + (type == EntityType::RoadEdge ? consts::lidarRoadEdgeOffset : consts::lidarRoadLineOffset);
```

Comment:
Why are there changes in the makeRoadEdge method for the lidar changes?

Reply:
The z heights are are offset so that road objects lie on different XY planes. This is because we dont want objects occluding other objects for lidar.